v1.1/admin-ui-overview-dashboard.md

```diff
@@ -0,0 +1,67 @@
+
+<div id="toc"></div>
+
+#### Cluster and Node View for the Time Series Graphs
+By default, the Time Series panel displays the cluster view, which shows the metrics for the entire cluster. 
+
+To access the node view that shows the metrics for an individual node, select the node from the **Graph** drop-down list.
+
+<img src="{{ 'images/admin_ui_select_node.png' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:40%"/>
+
+The **Overview** dashboard displays the following time series graphs:
+
+## SQL Queries
+<img src="{{ 'images/admin_ui_sql_queries.png' | relative_url }}" alt="CockroachDB Admin UI SQL Queries graph" style="border:1px solid #eee;max-width:100%" />
+
+In the node view, the SQL Queries metric depicts the current moving average, over the last 10 seconds, of the number of `SELECT`/`INSERT`/`UPDATE`/`DELETE` queries per second issued by SQL clients on the node. 
+
+In the cluster view, the graph shows the sum of the per-node averages; that is, an aggregate estimation of the current query load over the cluster, assuming the last 10 seconds of activity per node are representative of this load.
+
+## Service Latency: SQL, 99th percentile
+<img src="{{ 'images/admin_ui_service_latency_99_percentile.png' | relative_url }}" alt="CockroachDB Admin UI Service Latency graph" style="border:1px solid #eee;max-width:100%" />
+
+Service latency is calculated as the time between when the cluster receives a query and finishes executing the query. This time does not include returning results to the client. 
+
+In the node view, the graph displays the 99th [percentile](https://en.wikipedia.org/wiki/Percentile#The_normal_distribution_and_percentiles) of service latency for the node. 
+
+In the cluster view, the graph displays the 99th [percentile](https://en.wikipedia.org/wiki/Percentile#The_normal_distribution_and_percentiles) of service latency across all nodes in the cluster. 
+
+## Replicas per Node
+<img src="{{ 'images/admin_ui_replicas_per_node.png' | relative_url }}" alt="CockroachDB Admin UI Replicas per node graph" style="border:1px solid #eee;max-width:100%" />
+
+Ranges are subsets of your data, which are replicated to ensure survivability. Ranges are replicated to a configurable number of CockroachDB nodes. 
+
+In the node view, the graph displays the number of range replicas on the selected node.
+
+In the cluster view, the graph displays the number of range replicas on each node in the cluster. 
+
+You can [configure replication zones](configure-replication-zones.html) to set the number and location of replicas. You can monitor the changes in configuration using the Admin UI, as described in [Fault tolerance and recovery](demo-fault-tolerance-and-recovery.html).
```

Comment:
I don't think the link to the fault tolerance tutorial makes sense here. Let's just cut that sentence.

Reply:
Actually, I'd suggest replacing these last two paragraphs with this:

> For details about how to control the number and location of replicas, see Configure Replication Zones.

Once Sean's architecture docs are live (soon), we can add more relevant links.